libs/portals/admin/air-discount-scheme/src/lib/paths.ts

```diff
@@ -1,4 +1,5 @@
 export enum AirDiscountSchemePaths {
   Root = '/loftbru',
   CreateDiscount = '/loftbru/handvirkir-kodar',
+  SuperDiscount = '/loftbru/ofur-kodar',
```

Comment:
This could become confusing (discount VS super discount), should we add some inline docs that reference how this works (e.g. in notion)?